news/2023/2023-07-10-community-meeting-recap.md

```diff
@@ -0,0 +1,55 @@
+- Further internal discussion has led to the decision that leaderboards will be either wholly or partially wiped **just once** when the new scoring system is finalised, instead of multiple times.
+- ScoreV2 is now implemented as the current scoring system in osu!(lazer). It will be adjusted **one final time and then locked permanently**. Previous scores will be converted to this system.
+
+It's that last point that motivated this meeting. ScoreV2 has proven to be a better scoring system as it has been used for tournament play for years, but there may be issues that could be addressed. *What* adjustments to make, if any, is the big question.
+
+If you want to view the entire discussion, check out the VOD below:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/bocgaVISSfg" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Much of the discussion this time surrounded a long-debated topic: combo. Specifically, its significant — perhaps too significant — influence on score. These potential changes were discussed:
+
+- Directly decreasing the weight of the first few misses in the osu! game mode, since even one miss can have a huge impact on score.
+- Increasing the proportion in which accuracy affects score compared to combo in the osu! game mode. This was based on a [Twitter poll by peppy](https://twitter.com/ppy/status/1670681566547447808).
+- Drastically decreasing the value of high combos and aggressively scaling combo's influence based on accuracy, for all game modes. This change has been [submitted to GitHub](https://github.com/ppy/osu/pull/24166) already.
+
+There were some other issues brought up as well:
+
+- Hit values for slider ticks and some other judgement types currently don't match osu!(stable), which will be corrected.
+- Slider ends (which currently are almost completely optional to hit), slider hit accuracy and other changes to game mechanics have been contentious. These will need more discussion.
+- osu!mania PERFECT judgements are currently required for 100% accuracy, making it an incredibly difficult feat. Also to be discussed separately.
+
+In order to make sensible adjustments, focus should be spent on identifying problematic leaderboards and outlier scores that "feel wrong", to identify underlying issues and experiment with applying changes to get those scores closer to expectations.
+
+This is were *you* come in. We're looking for feedback on what issues the current ScoreV2 system has in order to nail down any remaining issues. The coming months will see the last steps taken to lock in scoring and make scores permanent in the new client.
+
+Keep in mind that there are as many opinions on scoring as there are players, and any adjustments will therefore focus on aligning things "close enough" to fix widely perceived issues, rather than debating over fine details.
+
+[Join the discussion in this forum thread!](LINK)
```

Comment:
Hmm, I get the intent but it feels like this won't work well unless a direction is given to the thread (ie. a very well formulated OP). Maybe best to leave it out for now as I don't think we have the bandwidth for this just yet, and there's already enough discussion happening on github.

Reply:
a similar type of thread has been made before fwiw https://osu.ppy.sh/community/forums/topics/1721243?n=1

note that inviting community discussion is the whole point of this post. i guess opening a thread isn't strictly necessary - it'd just mean that more opinions would be fired into the news post comments

by discussion on github i suppose you mean https://github.com/ppy/osu/discussions/24104, which is fair enough